source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -268,7 +268,7 @@
                 contentType: 'application/json; charset=utf-8',
                 data: JSON.stringify({ evaluationId: this.id }),
                 success: function (response) {
-                    getDashboardEvaluations(window.ViewData.period);
+                    window.location.reload();
```

Comment:
Not that I can recall :confused: 
Do you see any possible scenario that may be a problem?